SoftLayer/CLI/block/access/authorize.py

```diff
@@ -11,17 +11,17 @@
 
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
 @click.argument('volume_id')
-@click.option('--hardware-id', '-h', multiple=True,
-              help='The id of one SoftLayer_Hardware to authorize ' + MULTIPLE)
-@click.option('--virtual-id', '-v', multiple=True,
-              help='The id of one SoftLayer_Virtual_Guest to authorize ' + MULTIPLE)
+@click.option('--hardware-id', '-d', multiple=True,
+              help='The ID of one hardware server to authorize. ' + MULTIPLE)
 @click.option('--ip-address-id', '-i', multiple=True,
-              help='The id of one SoftLayer_Network_Subnet_IpAddress to authorize ' + MULTIPLE)
+              help='The ID of one IP address to authorize. ' + MULTIPLE)
```

Comment:
I prefer to have `SoftLayer_Network_Subnet_IpAddress` in this help since it refers to an ID, and I think users might get confused on what type of ID is required
```suggestion
              help='The ID of one SoftLayer_Network_Subnet_IpAddress to authorize. ' + MULTIPLE)
```

Reply:
Done